### DISCUSSION TITLE
2024/04/17/mt

### DISCUSSION THREAD

**Discussion by giscus** [wanghenshui/wanghenshui.github.io#147] [Q&A]
# 2024/04/17/mt

https://wanghenshui.github.io/2024/04/17/mt.html

**Comment by wanghenshui** [comment]
Categorizing How Distributed Databases Utilize Consensus Algorithms https://medium.com/@adamprout/categorizing-how-distributed-databases-utilize-consensus-algorithms-492c8ff9e916

这篇文章列举了两种观点

一种是 Consensus for Metadata (CfM) 只要保证元数据稳定，数据副本主备就行 比较省钱

一种是 Consensus for Data (CfD) 一致性算法同步，这种对网络分区要求高，需要跨地区部署，一致性要求高

其实大部分需求都是CfM场景，raft这种真用不上
